js/rooms.js

```diff
@@ -112,24 +112,22 @@
 			});
 		},
 		showRoomDeletedMessage: function(deleter) {
-			var message;
-
-			if (deleter) {
-				message = t('spreed', 'You have left the call');
-			} else {
-				message = t('spreed', 'This call has ended');
-			}
-
 			//Remove previous icon, avatar or link from emptycontent
 			var emptyContentIcon = document.getElementById("emptyContentIcon");
 			emptyContentIcon.removeAttribute("class");
 			emptyContentIcon.innerHTML = "";
 			$('#shareRoomInput').addClass('hidden');
 			$('#shareRoomClipboardButton').addClass('hidden');
 
-			$('#emptyContentIcon').addClass('icon-video-off');
-			$('#emptycontent h2').text(message);
-			$('#emptycontent p').text("");
+			if (deleter) {
+				$('#emptyContentIcon').addClass('icon-video');
```

Comment:
We use dashes, not camelcasing for classes. `#emptycontent-icon` would be better :)